data/systemd/yggdrasil.service.in

```diff
@@ -11,6 +11,11 @@ NotifyAccess=main
 WatchdogSec=300
 ExecStart=@bindir@/yggd
 PrivateTmp=true
+ExecStartPre=/bin/mkdir -p -m 740 /var/lib/yggdrasil
+ExecStartPre=/bin/chown -R yggd:yggd /var/lib/yggdrasil
+User=yggd
+Group=yggd
+DynamicUser=true
```

Comment:
The `DynamicUser=true` is the source of problems... When this option is set to `true`, then `/var/lib` is read only for whole service despite the `ExecStartPre` is executed as a root... Solution could be to create `yggd` user and group during installation of yggdrasil to the system as we do in #157.